int8 EMPTY = 0
int8 OCCUPY = 1
int8 ERROR = 2

Header header
int8 area_state
DoorState[] door_state